conn.go

```diff
@@ -388,6 +390,22 @@ var timestampConverter = valueConverterFunc(func(val interface{}) (driver.Value,
 	return nil, fmt.Errorf("%s: failed to convert %v (%T) into type time.Time", DriverName, val, val)
 })
 
+// objectIdConverter converts a value from the underlying json response into a objectid hex string.
+var objectIdConverter = valueConverterFunc(func(val interface{}) (driver.Value, error) {
+	if val == nil {
+		return nil, nil
+	}
+	if vv, ok := val.(string); ok {
+		data, err := base64.StdEncoding.DecodeString(vv)
+		if err != nil {
+			return nil, fmt.Errorf("%s: failed to convert %v into type objectId hex string", DriverName, val)
+		} else {
+			return fmt.Sprintf("%x", data), nil
```

Comment:
hex.EncodeToString(data) could be used here instead (it might be slightly faster)